cert.go

```diff
@@ -39,6 +40,34 @@ func init() {
 	userAndHostname += strings.TrimSpace(string(out))
 }
 
+// getFileName generate file name according to flags
+func (m *mkcert) getFileName(w string, args []string) (name string, err error) {
+	filename := strings.Replace(args[0], ":", "_", -1)
+	filename = strings.Replace(filename, "*", "_wildcard", -1)
```

Comment:
Are these replacements needed because of how we generate names?

Reply:
Yes, I do not want to modify the origin principle, if *FileFlag is not supplied, default filename will be generated, which means this PR has no effects on the users who do not care about the target file name.

Reply:
Ok, cool. Can we write a test around `getFileName`?

Reply:
> Ok, cool. Can we write a test around `getFileName`?

Test cases added. @adamdecaf